Sources/MapboxMaps/Foundation/Camera/CameraManager.swift

```diff
@@ -18,16 +18,8 @@ public class CameraManager {
         mapCameraOptions = newOptions
     }
 
-    /// Pointer HashTable for holding camera animators
-    internal var cameraAnimators = NSHashTable<CameraAnimator>.weakObjects()
-
-    /// List of animators currently alive
-    public var cameraAnimatorsList: [CameraAnimator] {
```

Comment:
just noticing that this moved to BaseMapView. I'm not necessarily opposed, but I want to discuss to make sure I understand the rationale.

Reply:
The `cameraAnimatorsList` can continue to live on `CameraManager`.. But the underlying hash table needs to be on `BaseMapView` so that it can call `update` on the animators.

Reply:
In a future where `BaseMapView` and `MapView` merge, this may no longer be necessary.